packages/keycloak-user-management/src/components/forms/UserForm/index.tsx

```diff
@@ -69,6 +69,29 @@ const UserForm: FC<UserFormProps> = (props: UserFormProps) => {
     form.setFieldsValue(initialValues);
   }, [form, initialValues]);
 
+  // get the status of user at mount
+  const [userEnabled, setUserEnabled] = useState<boolean>(initialValues.enabled ?? false);
+
+  // if user is disabled also disable practitioner
+  // else show default practitioner value
+  useEffect(() => {
+    if (!userEnabled) {
+      form.setFields([
+        {
+          name: 'active',
+          value: false,
+        },
+      ]);
+    } else {
+      form.setFields([
+        {
+          name: 'active',
+          value: initialValues.active,
+        },
+      ]);
+    }
+  }, [form, initialValues, userEnabled]);
```

Comment:
how about setting initialValues.value to dependency rather than whole initialValues object ?

Reply:
That's what setFields does. 

`form.setFields([
        {
          name: 'active',
          value: initialValues.active,
        },
      ]);`

This sets the 'active' field back to `initialValues.active`. I'm only using initialValues to get the initial 'active' value